src/tests/dictionaryConcepts/actions/dictionaryConcept.test.js

```diff
@@ -94,6 +99,26 @@ describe('Test suite for dictionary concept actions', () => {
     await fetchSourceConcepts('source', 'query');
   });
 
+  it('should query possible answer concepts', async () => {
+    const expectedAnswers = [{ id: 'Anwer 1' }, { id: 'Anwer 2' }];
+
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({ status: 200, response: expectedAnswers });
+    });
+    await queryAnswers('source', 'query');
```

Comment:
Where are you asserting or expecting?

Reply:
@dkayiwa this was an error. I have fixed it now.